src/test/kotlin/uk/gov/justice/digital/assessments/services/AssessmentUpdateServiceTablesTest.kt

```diff
@@ -1,358 +1,354 @@
+//
+// @ExtendWith(MockKExtension::class)
+// @DisplayName("Assessment Service Tables Tests")
+// class AssessmentUpdateServiceTablesTest {
+//   private val assessmentRepository: AssessmentRepository = mockk()
+//   private val episodeRepository: EpisodeRepository = mockk()
+//   private val riskPredictorsService: RiskPredictorsService = mockk()
+//   private val assessmentSchemaService: AssessmentSchemaService = mockk()
+//   private val oasysAssessmentUpdateService: OasysAssessmentUpdateService = mockk()
+//   private val authorService: AuthorService = mockk()
+//   private val auditService: AuditService = mockk()
+//   private val telemetryService: TelemetryService = mockk()
+//
+//   private val assessmentUpdateService = AssessmentUpdateService(
+//     assessmentRepository,
+//     episodeRepository,
+//     riskPredictorsService,
+//     oasysAssessmentUpdateService,
+//     authorService,
+//     auditService,
+//     telemetryService
+//   )
+//
+//   private val tableName = "test_table"
+//
+//   private val tableFields = QuestionSchemaEntities(
+//     listOf(
+//       QuestionSchemaEntity(
+//         questionSchemaId = 1L,
+//         questionSchemaUuid = UUID.randomUUID(),
+//         questionCode = "first_question",
+//       ),
+//       QuestionSchemaEntity(
+//         questionSchemaId = 2L,
+//         questionSchemaUuid = UUID.randomUUID(),
+//         questionCode = "second_question",
+//       )
+//     )
+//   )
+//
+//   private val assessmentUuid = UUID.randomUUID()
+//   private val episodeUuid = UUID.randomUUID()
+//   private val assessmentEpisode = AssessmentEpisodeEntity(
+//     episodeId = 1L,
+//     episodeUuid = episodeUuid,
+//     assessmentSchemaCode = AssessmentSchemaCode.ROSH,
+//     author = AuthorEntity(userId = "1", userName = "USER", userAuthSource = "source", userFullName = "full name"),
+//     assessment = AssessmentEntity()
+//   )
+//
+//   @BeforeEach
+//   fun setup() {
+//     every { assessmentSchemaService.toOasysAssessmentType(AssessmentSchemaCode.ROSH) } returns OasysAssessmentType.SHORT_FORM_PSR
+//     every { assessmentSchemaService.toOasysAssessmentType(AssessmentSchemaCode.RSR) } returns OasysAssessmentType.SOMETHING_IN_OASYS
+//     every { episodeRepository.save(any()) } returns null
+//     every { assessmentRepository.save(any()) } returns null
+//     every {
+//       oasysAssessmentUpdateService.updateOASysAssessment(any())
+//     } returns AssessmentEpisodeUpdateErrors()
+//     every { assessmentService.getCurrentEpisode(assessmentUuid) } returns assessmentEpisode
+//     every { assessmentService.getEpisode(assessmentUuid, episodeUuid) } returns assessmentEpisode
+//   }
+//
+//   @Test
+//   fun `adds an entry to a table for a given episode`() {
+//     val requestBody = UpdateAssessmentEpisodeDto(
+//       answers = mapOf(
+//         "first_question" to listOf("first_answer"),
+//         "second_question" to listOf("second_answer"),
+//       )
+//     )
+//
```

Comment:
we should really commit commented out code to the repo -  could you please delete.

Reply:
Thanks, had missed this file in the cleanup!